### ISSUE TITLE
[Feature Request]: Add animation to the icon at the very beginning

### ISSUE THREAD

**Issue by AnushkaS77** [manikumarreddyu/AgroTech-AI#132]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

once opened, an animation displays of the icon of agrotech ai like in netflix...to make its frontend more creative
animation include swirling of leaves with name displayed below it before the actual home page opens

### Use Case

make it look more interesting

### Priority

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by AnushkaS77** [comment]
@manikumarreddyu thankyou

**Comment by manikumarreddyu** [comment]
can you please let me know how much time do you need to complete this issue @AnushkaS77

**Comment by AnushkaS77** [comment]
Hello sir , the issue would be resolved by tomorrow.
Im thankful for your patience as im a new contributer and its my first
time.
If possible could i use this email as way of communication as im facing
some issues as well ?

**Comment by manikumarreddyu** [comment]
https://github.com/VaibhavArora314/StyleShare

https://style-share.vercel.app/

you can take reference from this website...and repo link provided..

connect me on linkedin https://www.linkedin.com/in/manikumarreddyu/

**Comment by manikumarreddyu** [comment]
Any updates..?

**Comment by AnushkaS77** [comment]
yes sir, the intro ive attached below. the problem im facing is that even
tho in the index.html file ive linked the reference to the video file...it
fails to show up in the local host preview
also the below is just a preview i would change to the colour as the
background colour of the home page . also do you want the same symbol as
the home page or should i change it ?
thank you

**Comment by manikumarreddyu** [comment]
You can take help from styleshare

**Comment by manikumarreddyu** [comment]
Inneed exactly like that...even you can put style share itself..no problem..
I will change later

**Comment by AnushkaS77** [comment]
the final intro with same bg colour

**Comment by manikumarreddyu** [comment]
Can you share me the video how it is working in your local .

**Comment by AnushkaS77** [comment]
its not woking in my local...displays nothing

**Comment by manikumarreddyu** [comment]
Okay then ...I'm closing this issue.im sorry for that

**Comment by AnushkaS77** [comment]
No problem, I understand.

**Comment by AnushkaS77** [comment]
sir its done...ill share the recording with you in a moment

**Comment by AnushkaS77** [comment]
its done ...can i commit changes??

**Comment by manikumarreddyu** [comment]
ok

**Comment by manikumarreddyu** [comment]
raise a PR

**Comment by manikumarreddyu** [comment]
https://www.linkedin.com/in/manikumarreddyu/
connect me here

**Comment by AnushkaS77** [comment]
sir but i havent yet created a pr...some connection issue pops up...
couldnt connect to server

**Comment by manikumarreddyu** [comment]
can you share me video of how it is working...here itself..
